### DISCUSSION TITLE
[Menu Entry Swapper] Ability to move "Escape" option to the top of (Corrupted) Gauntlet barriers

### DISCUSSION THREAD

**Discussion by OfficialTheosis** [runelite/runelite#15934] [Q&A] [answered]
Title says it all really. Would be nice to be able to do this for easier escapes from the Hunllef room.

Currently seem to be unable to customise the option order on these barriers (only when inside the Hunllef room) - no option to swap them when shift clicking as you can see here:
![escape](https://user-images.githubusercontent.com/26799016/202896487-fd54b527-7730-4f95-a2b8-ca6f3d5909b5.png)

Also can't seem to find a plugin to achieve this. Would this feature break any rules?

Feedback/info is much appreciated. Cheers.

**Comment by Nightfirecat** [comment, marked as answer]
This is the same issue as #9366, and will be available after our next client release.
